### ISSUE TITLE
when jupyter cell is running, show the elapsed time *while running*

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#5904]
Right now in the upper right we show the time it took to run a cell, once the cell is done running.  It would be very useful to show a sort of stopwatch as the cell runs, which would serve two purposes:

- it is another clear reminder that a cell is running, and
- it obviously tells you how long it has been running so far, which can be very helpful.

This is probably very easy to implement.

**Comment by williamstein** [comment]
@haraldschilly 's recent changes make it show the time when the eval started in a hover:

<img width="733" alt="image" src="https://github.com/user-attachments/assets/bb089d1d-aa22-4b09-939a-426f0d48ed8d">

That seems even better, since a constantly changing time would be annoying.  And it's already done.
